rustls/src/client/client_conn.rs

```diff
@@ -286,6 +286,12 @@ impl ClientConfig {
         }
     }
 
+    /// Return true if connections made with this `ClientConfig` will
+    /// operate in FIPS mode.
+    pub fn fips_mode(&self) -> bool {
```

Comment:
Naming nit: should we just call it `fips()` instead of `fips_mode()`? ISTM the `_mode` suffix doesn't really add value here.